### PR TITLE
First and last process

### PR DESCRIPTION
**Pull request by zansinergise**
https://git.sinergise.com/team-6/openeo-platform/-/issues/64

### CODE REVIEW

File: src/pg_to_evalscript/javascript_processes/last.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  if (data.length === 0) {
+    return null;
+  }
+
+  if (ignore_nodata) {
+    return [...data].reverse().find((x) => x !== null);
+  }
+
+  return [...data].reverse()[0];
```
Comment (dgostencnik) [inline src/pg_to_evalscript/javascript_processes/last.js:16]: I think there is more efficient way to get last element of an array.

File: tests/unit_tests/test_first.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+@pytest.fixture
+def first_process_code():
+    return load_process_code("first")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({'data': []}, None),
+        ({'data': [1,2,3]}, 1),
```
Comment (dgostencnik) [inline tests/unit_tests/test_first.py:17]: Can you also add test for an array will only null values

```
An array without non-null elements resolves always with null.
```

**Review by dgostencnik — CHANGES_REQUESTED** [review]

File: src/pg_to_evalscript/javascript_processes/last.js (hunk 1 above)
Comment (zansinergise) [reply src/pg_to_evalscript/javascript_processes/last.js]: Should be fixed

File: tests/unit_tests/test_first.py (hunk 2 above)
Comment (zansinergise) [reply tests/unit_tests/test_first.py:17]: Should be fixed

File: src/pg_to_evalscript/javascript_processes/last.js (hunk 1 above)
Comment (dgostencnik) [reply src/pg_to_evalscript/javascript_processes/last.js]: Sorry, I should have been more clear about it. What I had in mind is good old 
```
data[data.length-1] 
```
fftr

Comment (zansinergise) [reply src/pg_to_evalscript/javascript_processes/last.js]: According to google, `pop()` is supposed to be the fastest/most efficient way to get the last element of the array, so I went with this.
